guides/common/modules/con_migrating-project-to-a-new-el-system.adoc

```diff
@@ -1,10 +1,5 @@
-[id="migrating-project-to-el8_{context}"]
-ifdef::satellite[]
-= Migrating {ProjectName} to {RHEL} 8
-endif::[]
-ifndef::satellite[]
-= Migrating {ProjectName} to Enterprise Linux 8
-endif::[]
+[id="migrating-project-to-a-new-el-system_{context}"]
+= Migrating {Project} to a New Enterprise Linux System
```

Comment:
for Satellite, this probably should read "… to a New {RHEL} System"?
(We're trying to avoid RH naming for upstream, because it also runs on non-RHEL, but for Satellite it should be RHEL only)